.github/workflows/pr-jira-issue-transfer.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+name: External Authorship Check
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  check-author:
+    runs-on: self-hosted
+    steps:
+      - name: Check if PR is authored by an external contributor
+        run: |
+          # Fetch the author association of the PR
+          AUTHOR_ASSOCIATION=$(jq --raw-output .pull_request.author_association $GITHUB_EVENT_PATH)
+          
+          # Define the list of allowed author associations for team members
+          TEAM_ASSOCIATIONS=("MEMBER")
+          
+          # Check if the author association is not in the team list
+          if [[ ! " ${TEAM_ASSOCIATIONS[@]} " =~ " ${AUTHOR_ASSOCIATION} " ]]; then
+            - name: Login
+              uses: atlassian/gajira-login@v3        
+    
+            - name: Jira Create
+              uses: atlassian/gajira-create@v3
+              with:
+                  project: BP
+                  issuetype: 'Product Feature'
+                  summary: ${{ github.event.pulls.title }}
+                  description: "Github Pull Request Link: ${{ github.event.pulls.html_url}} \r\n ${{ github.event.pulls.body }}"
+                  fields: '{"labels":["GitHubReport"]}'

```

Comment:
This is nested inside a bash `run:` block. You can't mix YAML and bash syntax or use bash as a templating language.

Reply:
Uploaded the incorrect version,  corrected version and feel free to review